recipes/ufmt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ufmt" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ufmt-{{ version }}.tar.gz
+  sha256: 7e4518e13507984998f5c7c9a0d3f59fe014ddc213bdb5cae8d11a6a8935db8c
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
```suggestion
  number: 0
  noarch: python
```

Reply:
I needed to make this a non-noarch package as the PyPI wheel for Windows contains ufmt.exe which breaks the CI for windows. Happy to add back and skip Windows.

Reply:
It is probably the lack of the entrypoint.

Reply:
Specifying an entrypoint fixed the issues. Moved to noarch.